app/controllers/file_uploads_controller.rb

```diff
@@ -37,7 +37,7 @@ def download
 
   # POST /facilities/1/services/3/files
   def create
-    @file = @product.stored_files.new(params[:stored_file].merge(created_by: session_user.id))
+    @file = @product.stored_files.new(create_params.merge(created_by: session_user.id))
```

Comment:
Do we have a spec around this? I'm only about 50% confident that `merge` will keep `created_by` allowed.

Reply:
Added a spec. Looks like the merge is fine.